nidaqmx/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import nidaqmx
+from nidaqmx.grpc_session_options import GrpcSessionOptions
+import grpc
+
+@pytest.fixture(scope="module")
+def grpc_session_creation_kwargs():
+        channel = grpc.insecure_channel(f"localhost:31763")
+        grpc_options = GrpcSessionOptions(
+            grpc_channel = channel,
+            session_name = "NIDAQmxSession",
+        )
+        grpc_options = GrpcSessionOptions(channel, "")
+        return {'grpc_options': grpc_options}
+
+@pytest.fixture(scope="module")
+def library_session_creation_kwargs():
+    return {}
+
+@pytest.fixture(params = ("library_session_creation_kwargs", "grpc_session_creation_kwargs"), scope="module")
+def session_creation_kwargs(request):
+    return request.getfixturevalue(request.param)
```

Comment:
Good, if this works as expected, I think this will be lower touch than the approach that nimi-python uses (of subclassing the test class for library vs. grpc)

Reply:
Yes ,during my testing this approached worked perfectly fine.